arviz/plots/backends/matplotlib/forestplot.py

```diff
@@ -484,7 +505,7 @@ def treeplot(self, qlist, credible_interval):
 
     def ridgeplot(self, mult, ridgeplot_kind):
         """Get data for each ridgeplot for the variable."""
-        xvals, yvals, pdfs, colors = [], [], [], []
+        xvals, yvals, pdfs, pdfs_q, colors = [], [], [], [], []
```

Comment:
this could be 

    xvals = yvals = pdfs = pdfs_q = colors = []

but it is not that important, so no need to change it